source/css/sass/patterns/molecules/teaser.scss

```diff
@@ -0,0 +1,25 @@
+@import "../../mixins/spacing";
+@import "../../mixins/typography";
+@import "../../mixins/_decorations";
+
+.teaser-list__list {
+  @include padding(0, inline-start);
+}
+
+.teaser {
+
+  .teaser-list__item:not(:last-child) & {
+    @include margin($baselinegrid-space-small, block-end);
+    @include divider(block-end);
+  }
+
+}
+
+.teaser__header {
+  @include block-spacing($start: 0);
```

Comment:
Setting `$start` isn't need as it's the default. (`<header>` already has this too, so not needed?)